common/xrdp_constants.h

```diff
@@ -86,6 +86,11 @@
 #define MCS_SDRQ                       25 /* Send Data Request */
 #define MCS_SDIN                       26 /* Send Data Indication */
 
+/* xorgxrdp: frame capture interval (milliseconds) */
+#define RFX_FRAME_INTERVAL             32
+#define H264_FRAME_INTERVAL            16
+#define NORMAL_FRAME_INTERVAL          40
+
```

Comment:
Small point, but it might be worth calling these `DEFAULT_xxx_FRAME_INTERVAL` so that someone who is reading this code fragment realises they can be overridden.

Reply:
Makes sense.